Gemfile.lock

```diff
@@ -117,7 +117,7 @@ GEM
     minitest (5.14.4)
     msgpack (1.4.2)
     nio4r (2.5.8)
-    nokogiri (1.12.5-x86_64-darwin)
```

Comment:
I have a feeling this line would cause us some merge issues down the line. Should we get a TA to look at this?